File: Python/Produk.py
# library
from Shirt import Shirt

# membuat kelas produk
class Produk(Shirt):
    # konstruktor dengan parameter
    def __init__(self, id, name, brand, price):
        self.id = id
        self.name = name
        self.brand = brand
        self.price = price
    
    # set & get id
    def set_id(self, id):
        self.id = id
    
    def get_id(self):
        return self.id
    
    # set & get nama
    def set_name(self, name):
        self.name = name

    def get_name(self):
        return self.name
    
    # set & get brand
    def set_brand(self, brand):
        self.brand = brand

    def get_brand(self):
        return self.brand
    
    # set & get price
    def set_price(self, price):
        self.price = price
    
    def get_price(self):
        return self.price

File: PHP/Produk.php
<?php
// import file
require ('Shirt.php');

// deklarasi kelas produk
class Produk extends Shirt {
    // atribut private
    private $id = '';
    private $name = '';
    private $brand = '';
    private $price = '';

    // konstruktor
    public function __construct( $id = '', $name = '', $brand = '',$price='' ) {
        $this->id = $id;
        $this->name = $name;
        $this->brand = $brand;
        $this->price = $price;
    }

    // set & get id
    public function set_id($id) {
        $this->id = $id;
    }

    public function get_id() {
        return $this->id;
    }

    // set & get nama
    public function set_name($name) {
        $this->name = $name;
    }

    public function get_name() {
        return $this->name;
    }

    // set & get brand
    public function set_brand($brand) {
        $this->brand = $brand;
    }

    public function get_brand() {
        return $this->brand;
    }

    // set & get price
    public function set_price($price) {
        $this->price = $price;
    }

    public function get_price() {
        return $this->price;
    }

}

?>